### DISCUSSION TITLE
change shell

### DISCUSSION THREAD

**Discussion by purvjoshi04** [prasanthrangan/hyprdots#1973] [Q&A] [answered]
How can i change shell from fish to zsh the terminal does not show terminal themes with fish shell
![241109_10h44m15s_screenshot](https://github.com/user-attachments/assets/b817acc7-99f6-407d-a385-f8f3a5483c05)

**Comment by kRHYME7** [comment]
> How can i change shell from fish to zsh

`Hyde shell select ` 

>  the terminal does not show terminal themes with fish shell

As it should be. shell itself do not handle  terminal emulator theming. What are your expected behaviour?

**Reply by purvjoshi04** [reply]
I am looking for this but don't know what it's called, so I referred to it as a terminal theme. Apologies for the confusion.

<img width="1468" alt="Screenshot 2024-11-09 at 4 56 14 PM" src="https://github.com/user-attachments/assets/c907453d-a4ee-4679-9403-ca6c3d9c5479">

**Reply by kRHYME7** [reply]
ah I see. That's a pokemon script that shows pokemon ASCII art. It works on both Shell but Only zsh runs it on startup by default.

**Reply by purvjoshi04** [reply]
can you tell me how can i enable this script ? or do i need to install zsh ?

**Reply by kRHYME7** [reply]
@purvjoshi04  It is this command
![image](https://github.com/user-attachments/assets/b85f0a99-a230-4ec9-879d-63a44178f333)
```bash 
pokemon-colorscripts --no-title -r 1,3,6
```

It is in the `~/.zshrc` file because It will be executed  on `zsh` shell startup. 

in `fish shell ` , hyprdots repository did not add that command for startup therefore it is not activated on startup. 

>  or do i need to install zsh ?

I elaborated certain keywords for you to search to. Hope that helps.

**Reply by purvjoshi04** [reply]
i don't have this script how can i enable this script ?

**Reply by kRHYME7** [reply, marked as answer]
#1894  Try to read the discussion there are better alternatives.

**Reply by purvjoshi04** [reply]
Thank you @kRHYME7
